### PR TITLE
Add support for mounting persistent volumes

### PR DESCRIPTION
**Pull request by austinphilp**
Following Issue #79 and [this slack thread](https://gruntwork-community.slack.com/archives/CHH9Y3Z62/p1600730139039500) and I've added support for mounting a Kubernetes Persistent Volume onto a service pod using this helm chart.

I've added a persistentVolumes input to the `values.yaml` which will allow the user to define one or more already created persistent volumes to be mounted on the container. 

As this is a purely additive change, there should be no issues regarding backwards compatibility for consumers of this helm chart

### CODE REVIEW

File: charts/k8s-service/values.yaml [hunk 1]
```diff
@@ -372,6 +372,27 @@ additionalContainerEnv: {}
...
+# persistentVolumes:
+# - volumeName: example-pv
+#   mountPath: /mnt/myVol
+#   claimName: example-pv-claim
+# - volumeName: example-pv-2:
+#   mountPath: /mnt/myOtherVol
+#   claimName: example-pv2-claim
+#
+#
+persistentVolumes: []
```
Comment (yorinasub17) [inline charts/k8s-service/values.yaml:394]: It is better to organize this as a map, where the key is the volume name. The primary reason is that it makes it easier to override with `--set` if needed.

**Review by yorinasub17 — COMMENTED** [review]
Thanks for your contribution! The implementation LGTM, with one minor nit about the input variable type.

This also needs a template unit test so that we can verify the volumes are formatted correctly and will be parsed by the kubernetes API.

**Comment by austinphilp** [comment]
@yorinasub17 Thanks for taking a look, I'll go ahead and make the type change and add some template unit tests 👍

**Comment by yorinasub17** [comment]
Updates LGTM! Just kicked off a build and if it passes, we can merge this in!

**Review by yorinasub17 — APPROVED** [review]
Ok build passed, so this is good to go!

**Comment by yorinasub17** [comment]
Released as https://github.com/gruntwork-io/helm-kubernetes-services/releases/tag/v0.1.4

**Pull request merged**